testing/src/main/groovy/org/cristalise/restapi/test/PathAccessTest.groovy

```diff
@@ -0,0 +1,57 @@
+package org.cristalise.restapi.test
+
+import static io.restassured.RestAssured.*
+import static net.javacrumbs.jsonunit.JsonAssert.assertJsonEquals
+import static org.hamcrest.Matchers.*
+
+import org.cristalise.kernel.process.AbstractMain
+import org.cristalise.kernel.utils.Logger
+import org.json.JSONArray
+import org.junit.BeforeClass
+import org.junit.Test
+
+import groovy.transform.CompileStatic
+import io.restassured.http.ContentType
+import io.restassured.http.Cookie
+import io.restassured.response.Response
+
+@CompileStatic
+class PathAccessTest extends RestapiTestBase {
+    @Test
+    public void getBulkAliases() throws Exception {
+        login()
+        
+        Response aliasesResp =
+            given()
+                .accept(ContentType.JSON)
+                .cookie(cauthCookie)
+                .params('search', "['$userUuid', '00000000-0000-0000-0000-00000000000c', '00000000-0000-0000-0000-000000000410', '00000000-0000-0000-0000-000000010410']")
+            .when()
+                .get(apiUri+"/domain/aliases")
```

Comment:
If the uuids supplied are more than 50, this might fail. Shouldn't we use post instead?

Reply:
An Item can have many domain pathes (aka aliases) it is like a symbolic link in linux. We do not use this feature currently. The searchAliases(ip, 0, 50) searches aliases for one uuid, so we will not have a problem with that.

In restapi post means you are changing the resource, which is not the case, that is why it is a get.

Reply:
@kovax Then we might run out of the request size again, like we did in 3atech. We're sending 50 UUIDs in the request.
Also, regardless if it's GET or POST, we are actually doing RPC calls over HTTP, not "getting a resource" or "storing a resource". So our API is not _RESTFUL_ at all, therefore it doesn't make sense to enforce this one rule from the RESTFUL API rules, but not others.
@iangabrielsanchez Please implement it with POST also.

Reply:
The API was changed for 3atech for emergency reasons. but in principle they requirements is coming from a wrong design, those forms are far too big, Also our update script solution can be optimised to enable partial updates.

We are not in an emergency situation, therefore we can limit the number of UUIDs sent by the webui to 50 (up to 100), so I see no reason to implement it using POST.

cristal-ise http api is not an generic rpc, because each Item is a generic resource that is why it is called restapi. Using the REST Architecture style shall help us to create more consistent API that is easier to understand therefore easier to maintain and evolve. One of the principle of REST is that a GET operation shall not update any data in the system. If we keep that we will avoid future issues. 

On the other hand getAliases is not linked with an Item, which is an indication that we are doing something wrong. I will come up the proper solution.